packages/react/src/components/TextArea/TextArea.Skeleton.tsx

```diff
@@ -10,7 +10,21 @@ import React from 'react';
 import cx from 'classnames';
 import { usePrefix } from '../../internal/usePrefix';
 
-const TextAreaSkeleton = ({ hideLabel, className, ...rest }) => {
+type ExcludedAttributes = '';
+export interface TextAreaSkeletonProps
+  extends Omit<React.InputHTMLAttributes<HTMLDivElement>, ExcludedAttributes> {
```

Comment:
is the omit necessary here? looks like no Attributes are being omitted
```suggestion
export interface TextAreaSkeletonProps
  extends React.InputHTMLAttributes<HTMLDivElement> {
```

Reply:
You're correct, it isn't needed here. I've removed it!

Reply:
Hey @sierrawetmore looks like you removed it from `packages/react/src/components/TextArea/TextArea.tsx`, but I still see them on `packages/react/src/components/TextArea/TextArea.Skeleton.tsx`

Reply:
Oops! Fixed it.